widget/embedded/src/store/slices/data.ts

```diff
@@ -27,14 +27,15 @@ type TokenOptions = {
 export type FetchStatus = 'loading' | 'success' | 'failed';
 export interface DataSlice {
   _blockchainsMapByName: Map<string, BlockchainMeta>;
-  _tokens: Token[];
+  _tokensMapByHashToken: Map<string, Token>;
```

Comment:
I think `tokensMapByTokenHash` is a better name.

Reply:
fixed